monai/bundle/config_parser.py

```diff
@@ -116,6 +116,16 @@ def __init__(
     def __repr__(self):
         return f"{self.config}"
 
+    def __getattr__(self, id):
+        """
+        Get the config by id.
+
+        Args:
+            id: id to specify the expected position. See also :py:meth:`__getitem__`.
```

Comment:
Hi @Shadow-Devil ,

I think the doc-string should also be updated?
It's not `__getitem__ `.

Thanks.

Reply:
sorry, I forgot